### ISSUE TITLE
Building on Linux, “ImportError: cannot import name '_imaging'”

### ISSUE THREAD

**Issue by TinaRussell** [asiekierka/zeta#16]
I tried to build Zeta on my Linux (Ubuntu MATE 20.04) machine, following the instructions, only to get this error during the make process:
```
Traceback (most recent call last):
  File "../tools/font2raw.py", line 16, in <module>
    from PIL import Image
  File "/usr/lib/python3/dist-packages/PIL/Image.py", line 69, in <module>
    from . import _imaging as core
ImportError: cannot import name '_imaging'
```
I have libsdl2-dev installed, as well as python3-pil. I’m not sure what is going wrong. Indeed, if I start python3 from the command line and try entering `from PIL import Image`, I get the same message:
```
>>> from PIL import Image
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3/dist-packages/PIL/Image.py", line 69, in <module>
    from . import _imaging as core
ImportError: cannot import name '_imaging'
```
So I guess there’s a problem with PIL, but I don’t know what it is or how to get around it :confused: I tried searching for solutions to this problem, but everything I read (for instance, [here](https://stackoverflow.com/questions/25340698/importerror-cannot-import-name-imaging)) concerns something called Pillow, which is, I guess, a fork of PIL…? …and from there it kind of all goes over my head. (For instance, many responses seem to suggest installing or upgrading Pillow via pip, but I don’t want to be installing something via pip if it might conflict with what I have already installed via apt, unless I actually know what I’m doing, which I don’t. :sweat_smile:)

**Comment by asiekierka** [comment]
PIL is the Python Imaging Library. As its original maintainer hasn't worked on it in many years, it has been forked into a project named Pillow. Ubuntu 20.04 bundles Pillow 7.0.0 in the `python3-pil` package.

I've done some research on it and it seems the commonly accepted solution - including on [Pillow's official issue tracker](https://github.com/python-pillow/Pillow/issues/4982) - is to uninstall the apt version of Pillow and install the pip version.

**Comment by TinaRussell** [comment]
Okay, thanks. Unforunately, uninstalling Pillow via apt would uninstall a lot of other packages :confused: I’ve filed a bug with the apt package: https://bugs.launchpad.net/ubuntu/+source/pillow/+bug/1993190

**Comment by TinaRussell** [comment]
I figured out the problem. For some reason, there was a symlink (`/usr/local/bin/python3/`) that was pointing to an old version of Python (`/usr/bin/python3.6`) that was still having this problem. For some reason, removing that symlink wasn’t enough (the compiler was still looking for `/usr/local/bin/python3`), so I made a new symlink in its place that pointed to the regular `/usr/bin/python3` (which is a symlink to `/etc/alternatives/python3`, which is a symlink to `/usr/bin/python3.10`, at least on my system). Now the build process works fine.
